blueocean-i18n/src/main/java/io/jenkins/blueocean/i18n/BlueI18n.java

```diff
@@ -136,12 +136,19 @@ public HttpResponse doDynamic(StaplerRequest request) {
     private JSONObject getBundle(BundleParams bundleParams, Locale locale) {
         PluginWrapper plugin = bundleParams.getPlugin();
 
-        if (plugin == null) {
-            return null;
-        }
 
         try {
-            ResourceBundle resourceBundle = ResourceBundle.getBundle(bundleParams.bundleName, locale, plugin.classLoader);
+            ResourceBundle resourceBundle;
+            if (plugin == null) {
+                Jenkins jenkins = Jenkins.getInstance();
+                if (jenkins!=null && jenkins.pluginManager!=null && jenkins.pluginManager.uberClassLoader!=null) {
```

Comment:
As I said to @scherler ... whenever I see code with `uberClassLoader`, a big flashing light comes on ... "uber hack in progress here"